src/pspm_check_model.m

```diff
@@ -201,11 +201,11 @@
 else
   if ~isempty(model.timing)
     if ~iscell(model.timing) || ...
-      strcmpi(modeltype, 'dcm') && ~iscell(model.timing{1})
+      (strcmpi(modeltype, 'dcm') && ~iscell(model.timing{1}) && ~ischar(model.timing{1}))
       % for DCM, model.timing is either a file name or a cell array of
       % events, or a cell array of file names or cell arrays, so we need to
```

Comment:
I am a little confused about this... It says `model.timing` can be a cell array of filenames, but it cannot be a cell array of char. I think a filename is a char..?

Reply:
Hi Teddy, the comment does not actually say that "`model.timing` cannot be a cell array of char". Of course it can. The comment only says that we need to find the case where `model.timing` is a cell array, but NOT a cell array of char or of cell arrays (which are both perfectly permissible)